torch/csrc/jit/passes/quantization.cpp

```diff
@@ -445,6 +486,11 @@ class InsertObserversHelper {
   // the middle of the ops that are supposed to be fused, e.g.
   // the output value of conv in the conv - relu pattern
   std::unordered_set<Value*> values_to_skip_;
+  std::unordered_set<Graph*> visited_graph_of_observer_map_;
+  std::unordered_map<Value*, script::Module> observer_for_value_;
+  // Map from values from callsite into the values in the CallMethod graph
+  std::unordered_map<Value*, std::vector<Value*>> boundary_value_map_;
```

Comment:
It's probably better to use `unordered_set` instead of `vector` here. We never need the values in it in any particular order.